### PR TITLE
Test c controller

### PR DESCRIPTION
**Pull request by TunVunDerBunn**
This new function was implemented in the controller to debug the hardware

### CODE REVIEW

File: src/c_implementation/controller/controller_const.h [hunk 1]
```diff
@@ -22,7 +22,10 @@
 #define M_AW 0.005                                                                              // -
 #define SAFETY_COUNTER_THRESHOLD (int)(fmaxf(ceil(CONTROLLER_SAMPLING_FREQ/20.0), 2))           // -
 #define DELTA_T 1.0f / CONTROLLER_SAMPLING_FREQ                                                 // s
-#define TARGET_AGOGEE 1450.0                                                                   // m
+#define TARGET_AGOGEE 1450.0                                                                    // m
+#define TEST_CONTROLLER_USE_VELOCITY true                                                      // -
+#define TEST_CONTROLLER_MAX_VELOCITY 172.0f                                                       // m/s
+#define TEST_CONTROLLER_MAX_ALTITUDE 1450.0f                                                       // m
```
Comment (mstoelzle) [inline src/c_implementation/controller/controller_const.h:28]: Shouldnt this be the altitude of Hönggerberg?

File: src/c_implementation/controller/controller_const.h [hunk 2]
```diff
@@ -22,7 +22,10 @@
 #define M_AW 0.005                                                                              // -
 #define SAFETY_COUNTER_THRESHOLD (int)(fmaxf(ceil(CONTROLLER_SAMPLING_FREQ/20.0), 2))           // -
 #define DELTA_T 1.0f / CONTROLLER_SAMPLING_FREQ                                                 // s
-#define TARGET_AGOGEE 1450.0                                                                   // m
+#define TARGET_AGOGEE 1450.0                                                                    // m
+#define TEST_CONTROLLER_USE_VELOCITY true                                                      // -
+#define TEST_CONTROLLER_MAX_VELOCITY 172.0f                                                       // m/s
```
Comment (mstoelzle) [inline src/c_implementation/controller/controller_const.h:27]: We will never reach 172 m/s during the test, right?

File: src/c_implementation/controller/task_controller.c [hunk 3]
```diff
@@ -48,7 +48,9 @@ void vTaskController(void *argument) {
 			}
 		}
 
-		compute_control_input(&control_data, &current_flight_phase_detection);
+		/** TAKE CARE THE TEST CONTROLLER IS ACTIVE!!!!! **/ // todo: changed it if required
+        //compute_control_input(&control_data, &current_flight_phase_detection);
```
Comment (mstoelzle) [inline src/c_implementation/controller/task_controller.c:52]: Make the normal control input function active by default. This is safer for the real flight (to not accidentally forget changing the option)

File: src/c_implementation/controller/controller_const.h (hunk 1 above)
Comment (TunVunDerBunn) [reply src/c_implementation/controller/controller_const.h:28]: No but it should be only 4 m or something like this, we might need to adapted the env to the altitude of Hönggerberg

File: src/c_implementation/controller/controller_const.h (hunk 2 above)
Comment (TunVunDerBunn) [reply src/c_implementation/controller/controller_const.h:27]: yes, sorry I forgot to change the values back to the test values after I validated it

File: src/c_implementation/controller/task_controller.c (hunk 3 above)
Comment (TunVunDerBunn) [reply src/c_implementation/controller/task_controller.c:52]: is the new implementation better?

**Pull request merged**